pkgs/development/python-modules/aiolyric/default.nix

```diff
@@ -24,7 +25,15 @@ buildPythonPackage rec {
     hash = "sha256-pN/F4Rdov06sm1yfJQEzmWyujWVeVU+bNGGkgnN4jYw=";
   };
 
-  build-system = [ setuptools ];
+  postPatch = ''
+    substituteInPlace requirements_setup.txt \
+      --replace-fail "==" ">="
+  '';
```

Comment:
This file does not exist in this version.

Reply:
It does, but not in the one pinned by home-assistant.